resources/language/German/strings.po

```diff
@@ -474,7 +474,7 @@ msgstr "Empfange Wiedergabefortschritt der Episoden von Trakt"
 
 msgctxt "#32120"
 msgid "Parsing episode playback progress %i of %i from Trakt"
-msgstr "Verarbeite Wiedergabefortschritt der Episoden %i of %i von Trakt"
+msgstr "Verarbeite Wiedergabefortschritt der Episoden %i von %i von Trakt"
```

Comment:
It's also my favourite of these two. I was worried about the possible width though - that's the reason for version two! Glad I could possibly help and again, sorry for my pull request.